### PR TITLE
Kotlin 1.6.20

### PR DESCRIPTION
**Pull request by rickclephas**

### CODE REVIEW

File: kmp-nativecoroutines-compiler/src/main/kotlin/com/rickclephas/kmp/nativecoroutines/compiler/KmpNativeCoroutinesSyntheticResolveExtension.kt [hunk 1]
```diff
@@ -109,7 +109,8 @@ internal class KmpNativeCoroutinesSyntheticResolveExtension(
         val type = thisDescriptor.module.getExpandedNativeFlowType(valueType)
         return createPropertyDescriptor(thisDescriptor, coroutinesPropertyDescriptor.visibility,
             name, type, coroutinesPropertyDescriptor.dispatchReceiverParameter,
-            coroutinesPropertyDescriptor.extensionReceiverParameter
+            coroutinesPropertyDescriptor.extensionReceiverParameter,
+            coroutinesPropertyDescriptor.contextReceiverParameters
```
Comment (rickclephas) [inline kmp-nativecoroutines-compiler/src/main/kotlin/com/rickclephas/kmp/nativecoroutines/compiler/KmpNativeCoroutinesSyntheticResolveExtension.kt:113]: Support should also be added to the IRTransformer

Comment (rickclephas) [reply kmp-nativecoroutines-compiler/src/main/kotlin/com/rickclephas/kmp/nativecoroutines/compiler/KmpNativeCoroutinesSyntheticResolveExtension.kt:113]: Turns out these are part of the `valueParameters` so no changes are required in the IrTransformer

**Comment by rickclephas** [comment]
Everything seems to be fine according to Github Actions but I am having some issues building this locally.
https://youtrack.jetbrains.com/issue/KT-51293

**Comment by hfhbd** [comment]
What about https://youtrack.jetbrains.com/issue/KT-47399 support? This maps `return KotlinUnit.shared` to `return Void`, if needed.

**Comment by rickclephas** [comment]
> What about https://youtrack.jetbrains.com/issue/KT-47399 support? This maps `return KotlinUnit.shared` to `return Void`, if needed.

Thanks @hfhbd! I hadn't seen that one just yet.
Anyway this shouldn't effect KMP-NativeCoroutines as it doesn't expose any suspend functions 🙂.

**Pull request merged**